cli/command/trust/sign.go

```diff
@@ -19,19 +19,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signOptions struct {
+	local bool
+}
+
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
+	options := signOptions{}
 	cmd := &cobra.Command{
 		Use:   "sign [OPTIONS] IMAGE:TAG",
 		Short: "Sign an image",
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return signImage(dockerCli, args[0])
+			return signImage(dockerCli, args[0], options)
 		},
 	}
+	flags := cmd.Flags()
+	flags.BoolVarP(&options.local, "local", "l", false, "Sign a local image")
```

Comment:
can we make this clearer that it means that multisigning won't work?  Though actually, I suppose it's possible that your local image is the same as the already-signed version...so I'm not sure how to best phrase this though.  I'll think more about this.